odoo/models.py

```diff
@@ -1695,6 +1696,31 @@ def search_count(self, args):
         res = self.search(args, count=True)
         return res if isinstance(res, int) else len(res)
 
+    @classmethod
+    @functools.lru_cache
+    def _cached_search(cls, model, args, limit):
+        # print(args)
+        return model.search(list(args), limit=limit)
```

Comment:
FYI, this cache is shared across databases.
In other words, a request on one database will get the results of a search from another database!

Reply:
:facepalm:

Reply:
I removed the commit, but it is just a matter of adding a key for the database in the cache